src/core/index.js

```diff
@@ -1,5 +1,8 @@
 import { documentReady } from './util/dom';
 import { Docsify } from './Docsify';
+import initGlobalAPI from './global-api';
+
+initGlobalAPI();
```

Comment:
This is a small change: I moved `initGlobalAPI` call from `Docsify.js` to `index.js` so that importing `Docsify.js` has no side-effects.